mods/ra/rules/player.yaml

```diff
@@ -18,6 +18,7 @@ Player:
 		LowPowerSlowdown: 3
 		QueuedAudio: Building
 		SpeedUp: True
+		BuildTimeSpeedReduction: 100, 75, 60, 50
```

Comment:
The irony of this is, players remembered the 4-series WF queue values in the playtest compared to the 7-part default value. I'm all for streamlining features for ease of understanding by the players but in this case comparatively to the default value the difference would imo be negligable.